tests/test_rpmutils.py

```diff
@@ -108,6 +109,25 @@ def test_labelCompare(self):
         self.assertEqual(rpmLabelCompare(b, g), 1)
         self.assertEqual(labelCompare(b, f), -1)
 
+    def test_cpaas_label_compare(self):
```

Comment:
https://travis-ci.org/github/release-depot/toolchest/jobs/741713517

Need an extra line to clear up the flake8 issue above.

Reply:
Thanks for the fix.